From b3031fa2b3e407c1db565424f5e2d49fda6c8f27 Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Sun, 6 May 2018 09:14:10 +0000
Subject: [PATCH] passworddialog.cpp: include QIcon explicitely

5.11 Qt headers do not indirectly include QIcon resulting in compile
time errors like:
| passworddialog.cpp:44:33: error: incomplete type 'QIcon' used in nested name specifier

Upstream-Status: Submitted [https://github.com/lxqt/lxqt-sudo/pull/34]

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 passworddialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/passworddialog.cpp b/passworddialog.cpp
index 7b6de2a..e7a691d 100644
--- a/passworddialog.cpp
+++ b/passworddialog.cpp
@@ -27,6 +27,7 @@
 
 #include "passworddialog.h"
 #include "ui_passworddialog.h"
+#include <QIcon>
 
 PasswordDialog::PasswordDialog(QStringList argv
         , QWidget * parent/* = 0*/
-- 
2.13.6

